src/flavors/defaultflavor/config.json

```diff
@@ -1,4 +1,26 @@
 {
+  "datasets": [
+    {
+      "dataset_slug": "demo",
+      "url": "https://dev-api.heyduwamish.org/api/v2/smartercleanup/datasets/demo",
+      "anonymous_permissions": [
```

Comment:
These `anonymous_permissions` essentially mirror origin permissions on the API (or should at least). Eventually I think these could be derived from the API in the same way group permissions are in this PR.